apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -118,72 +176,180 @@ export default class ExportHTMLModal extends Vue {
   // If we are exporting a profile we can remove the test/results table
   @Prop({type: String, required: true}) readonly fileType!: string;
 
+  // Generated injectable HTML for icons
+  private iconHTMLStore = {
+    // Passed; green
+    circleCheck: this.iconDatatoSVG(
+      mdiCheckCircle,
+      'rgb(76, 176, 79)',
```

Comment:
check out the colors in the color hack file to make sure they're the same.  ideally we can just grab them from there but at least we should make a reference here and there that the colors must stay aligned

Reply:
Changed as requested.